### PR TITLE
Automation tenant storahe

### PR DESCRIPTION
**Pull request by katperich**
* need to update resolve task article with i**nformation on post-resolving + ignoring the task**
* needs **update of email screenshot** when available

### CODE REVIEW

File: governance-and-automation/automated-workflows/tenant-storage-admin.md [hunk 1]
```diff
@@ -19,7 +19,18 @@ The **Edit Policy** pop-up dialog opens where you can:
 * **Select the Severity level (2)**
   * The severity level set by default for this policy is High
   * You can select between three levels of severity: **High, Medium, or Low**
+* **Click the toggle to turn on automation (3)** for the policy
```
Comment (matejper) [inline governance-and-automation/automated-workflows/tenant-storage-admin.md]: automation je automatski uključen inicijalno i šalje se defaultno za Point admine

File: governance-and-automation/security-compliance-checks/tenant-storage.md [hunk 2]
```diff
@@ -9,9 +9,9 @@ Syskit Point mitigates the risk of running out of storage based on the settings
...
+The Tenant Storage policy is **a tenant-wide policy, and it can be automated**. When automation is enabled, Syskit Point creates tasks and sends emails to users as defined in the policy settings. To learn more, take a look at the [Tenant Storage Limit policy article](../../governance-and-automation/automated-workflows/tenant-storage-admin.md).
 
-When a tenant storage limit is exceeded, Syskit Point **detects the policy vulnerability** and shows it on the Security and Compliance dashboard. **This means that Syskit Point detects a vulnerability on a workspace** based on the applied policy, but it does not create tasks or send any emails. 
+By default, the automation is turned off for this policy, but when the tenant storage limit is exceeded, Syskit Point **detects the policy vulnerability** and shows it on the Security and Compliance dashboard. **This means that Syskit Point detects a vulnerability on a workspace** based on the applied policy, but it does not create tasks or send any emails to workspace owners.
```
Comment (matejper) [inline governance-and-automation/security-compliance-checks/tenant-storage.md]: treba ispraviti da je automation defualtno upaljen odmah

**Review by matejper — APPROVED** [review]

File: governance-and-automation/automated-workflows/tenant-storage-admin.md [hunk 3]
```diff
@@ -19,7 +19,15 @@ The **Edit Policy** pop-up dialog opens where you can:
 * **Select the Severity level (2)**
   * The severity level set by default for this policy is High
   * You can select between three levels of severity: **High, Medium, or Low**
+* By default, **automation for this policy is turned on** but it can be turned off by clicking the toggle (3)
+  * By default, Syskit Point admins are notified about storage consumptions but you can also select Custom Recipients
```
Comment (imesaric) [inline governance-and-automation/automated-workflows/tenant-storage-admin.md]: storage usage umjesto storage consumptions

File: governance-and-automation/security-compliance-checks/tenant-storage.md [hunk 4]
```diff
@@ -9,9 +9,9 @@ Syskit Point mitigates the risk of running out of storage based on the settings
...
+The Tenant Storage policy is **a tenant-wide policy, and it can be automated**. When automation is enabled, Syskit Point creates tasks and sends emails to users as defined in the policy settings. To learn more, take a look at the [Tenant Storage Limit policy article](../../governance-and-automation/automated-workflows/tenant-storage-admin.md).
 
-When a tenant storage limit is exceeded, Syskit Point **detects the policy vulnerability** and shows it on the Security and Compliance dashboard. **This means that Syskit Point detects a vulnerability on a workspace** based on the applied policy, but it does not create tasks or send any emails. 
+By default, the automation is turned on for this policy. If the policy automation is turned off, when the tenant storage limit is exceeded, Syskit Point still **detects the policy vulnerability** and shows it on the Security and Compliance dashboard. **This means that Syskit Point detects a vulnerability on a workspace** based on the applied policy, but if the policy automation is turned off, it does not create tasks or send any emails to workspace owners.
```
Comment (imesaric) [inline governance-and-automation/security-compliance-checks/tenant-storage.md]: send any emails to Syskit Point Administrators. ovdje se radi o adminima više nego ownerima

**Pull request merged**